src/components/Application/Navigation.tsx

```diff
@@ -85,18 +87,51 @@ const Navigation: React.FC<NavigationProps> = ({
       return
     }
 
-    // Use validationMethod to check if can change to page (on linear application) OR
-    // display current page with strict validation
+    // Use validationMethod to check if can change to page (on linear
+    // application) OR display current page with strict validation
     requestRevalidation(({ firstStrictInvalidPage, setStrictSectionPage }: MethodToCallProps) => {
       if (
         firstStrictInvalidPage !== null &&
         current.sectionCode === firstStrictInvalidPage.sectionCode &&
         current.pageNumber === firstStrictInvalidPage.pageNumber
+      )
+        setStrictSectionPage(firstStrictInvalidPage)
+      else {
+        setStrictSectionPage(null)
+        sendToPage(nextSectionPage)
+      }
+    })
+  }
```

Comment:
This is not added, Git just thinks it is because it's the same as the actually added one at the bottom of this block (which is *not* showing as added).